src/_packagedcode/pypi.py

```diff
@@ -22,15 +22,14 @@
 import zipfile
 
 import dparse2
-import packaging
+import packvers as packaging
 import pip_requirements_parser
-import pkginfo2
```

Comment:
Why did you remove thin import? Please reinstate

Reply:
Added : )

Reply:
But you moved it to a spot that's not in the correct sort order and should be right after the `import pip_requirements_parser`

Reply:
Done with the order :") , but wanted to know the reason.

Reply:
we sort package imports like isort would do

Reply:
Please just keep this this way:
```
import pip_requirements_parser
import pkginfo2
```

Reply:
Ohk got it ✨